narwhal/primary/src/metrics.rs

```diff
@@ -73,10 +73,6 @@ pub struct PrimaryChannelMetrics {
     pub tx_primary_messages: IntGauge,
     /// occupancy of the channel from the `primary::PrimaryReceiverHandler` to the `primary::Helper`
     pub tx_helper_requests: IntGauge,
-    /// occupancy of the channel from the `primary::ConsensusAPIGrpc` (when external consensus is being
-    /// used) & `executor::Subscriber` (when internal consensus, ex Bullshark, is being used)  to
-    /// the `primary::BlockWaiter`.
-    pub tx_get_block_commands: IntGauge,
     /// occupancy of the channel from the `primary::WorkerReceiverHandler` to the `primary::BlockWaiter`
     pub tx_batches: IntGauge,
```

Comment:
We can also remove the `tx_batches` and `tx_batches_total` metrics as well - those were `block_waiter` specific ones